compiler/luci/service/src/CircleShapeInferenceHelper.h

```diff
@@ -48,9 +48,8 @@ loco::TensorShape circle_shape(const luci::CircleNode *node);
 // Throw an exception if x and y are not broadcastable.
 loco::TensorShape broadcast_shape(const loco::TensorShape &x, const loco::TensorShape &y);
 
-// Return shape of pad ops using paddings.
-loco::TensorShape pad_shape(const loco::TensorShape &input_shape,
-                            const luci::CircleConst *paddings);
+// Return shape of pad ops using const paddings.
+loco::TensorShape pad_shape(const loco::TensorShape &input_shape, const luci::CircleNode *paddings);
```

Comment:
How about this? 
At first glance, I feel hard to catch the meaning of `const paddings`. 

```suggestion
// Return shape of pad ops using padding
// If paddings type is not a CircleConst, return the shape filled with unknown dimensions. 
loco::TensorShape pad_shape(const loco::TensorShape &input_shape, const luci::CircleNode *paddings);
```

Reply:
> How about this? At first glance, I feel hard to catch the meaning of `const paddings`.


Nice point!
I miss the annotation was that of `const`.
Now we can support `non-const paddings` with this PR, it should be changed.
I'll fix it.

Reply:
I thin there is some misuderstanding of `const`.
`const` of `const luci::CircleNode *paddings` has nothing to do with `CircleConst`.
Instead, implementation of this method(function) CANNOT modify `paddings`.

Reply:
> I think there is some misuderstanding of `const`.

IHMO `const` she mentioned is not about 
`const luci::CircleNode *paddings`. It is from annotation of function, `Return shape of pad ops using padding`.

This annotation is written before this PR which `TODO : support non-const paddings` , so with this PR this annotation can be changed as her suggestions.

Reply:
Here's a refined version of your sentence:

I think I misunderstood his earlier point. 😂

There seems to be some confusion about the phrase `// Return shape of pad ops using padding`:

1. Interpreting `const` as referring to `const luci::CircleNode *paddings`
2. Interpreting it as meaning the padding node is constant

My original intention is the second interpretation, as we only support `paddings` type with `CricleConst`.

How about clarifying the sentence by revising it as follows?

```suggestion
// Return shape of pad ops using paddings.
// If paddings is not static,  return the shape filled with unknown dimensions. 
loco::TensorShape pad_shape(const loco::TensorShape &input_shape, const luci::CircleNode *paddings);
```

Reply:
looks good to me :)